guides/writing_events.md

```diff
@@ -68,8 +69,64 @@ around the range of 10 and 100 KB), Spear takes conceptual breaks somewhat
 often during large requests. This allows Spear to efficiently multiplex large
 writes with large reads and subscriptions.
 
-## Batching
+## Increasing append throughput with `Spear.append_batch/5`
 
-When appending multiple events, Spear will fit as many messages as possible
-into the same HTTP2 DATA frame. This is valid according to the gRPC
-specification and has the potential to improve performance.
+EventStoreDB v21.6.0+'s BatchAppend feature is designed to improve performance
+of appending large numbers of events. The protobuf and gRPC service definitions
+give BatchAppend an advantage over the standard Append RPC both in terms
+of encoded message size (and therefore bytes over the network) and number of
+HTTP/2 requests.
+
+`Spear.append/4` is a simpler interface for appends and is a reasonable
+default. `Spear.append_batch/5` is more cumbersome to use but is also more
+powerful. In general, `Spear.append_batch/5` should be preferred when
+writing large numbers of events and when append throughput is critical.
```

Comment:
Could be a handy tool for writing a really high throughput shovel, but only usable when the sink eventstore is v21.6.0+

Reply:
I definitely saw the shovel example. I don't make the connection between shovel and `throughput is critical`.

Reply:
ah yeah for the throughput being important thing I could imagine some really fast-moving or bursty aggregate that would benefit from the speed boost here

frankly I don't think batch-append is all that useful for our needs but I could imagine some faster moving domains where it might come in handy